xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -33,6 +33,8 @@ struct Slashlink:
     
     var id: String { description }
     
+    static let profileSlug = "_profile_"
```

Comment:
This will be changed depending on the result of https://github.com/subconsciousnetwork/subconscious/issues/480